### PR TITLE
CCITCARBON-269 CCITCARBON-278 Added get_message_headers method

### PR DESCRIPTION
**Pull request by rujutashinde**

### CODE REVIEW

File: teflo_webhooks_notification_plugin/webhooks_notification_plugin.py [hunk 1]
```diff
@@ -140,6 +149,6 @@ def __init__(self, notification):
 
         super(GchatNotificationPlugin, self).__init__(notification=notification)
 
-        self.url = self.url = self.creds_params.get('gchat_url', None)
```
Comment (JunqiZhang0) [inline teflo_webhooks_notification_plugin/webhooks_notification_plugin.py:143]: Was it a typo here before?

**Review by JunqiZhang0 — APPROVED** [review]
LGTM

File: teflo_webhooks_notification_plugin/webhooks_notification_plugin.py (hunk 1 above)
Comment (rujutashinde) [reply teflo_webhooks_notification_plugin/webhooks_notification_plugin.py:143]: yes this was a typo before, i guess it got overlooked as it did not caused any errors. it looks like a copy paste issue by me :)

**Comment by rujutashinde** [comment]
@JunqiZhang0 I have modified this PR , trying to take care of two things which are related. Please go through the PR again.
I apologize for the extra run

File: tests/functional/test_webhook_notification.py [hunk 2]
```diff
@@ -159,22 +173,39 @@ def test_notify_method(mock_method2, mock_method1, slack_notification_plugin):
...
+    def test_get_message_headers(gc_notification_plugin):
+        """To test when no custom headers or username/pass are provided message_headers is created correctly"""
+        op = gc_notification_plugin.get_message_headers()
+        assert isinstance(op, dict)
+        assert 'Authorization' not in op.keys()
+        assert len(op) == 1
+
 
+    @staticmethod
+    def test_get_message_headers_1(webhook_notification_plugin):
```
Comment (JunqiZhang0) [inline tests/functional/test_webhook_notification.py]: Can the name of this test bit more intuitive? Like what use case is it testing?

File: tests/functional/test_webhook_notification.py [hunk 3]
```diff
@@ -159,22 +173,39 @@ def test_notify_method(mock_method2, mock_method1, slack_notification_plugin):
...
+        """To test when custom headers or username/pass are provided message_headers is created correctly"""
+        op = webhook_notification_plugin.get_message_headers()
+        assert isinstance(op, dict)
+        assert 'tenant' in op.keys()
+        assert 'Authorization' in op.keys()
 
+    @staticmethod
+    def test_get_message_headers_2(webhook_notification_plugin):
```
Comment (JunqiZhang0) [inline tests/functional/test_webhook_notification.py]: Maybe we could change this to a more intuitive name?

**Review by JunqiZhang0 — APPROVED** [review]
LGTM

File: tests/functional/test_webhook_notification.py (hunk 2 above)
Comment (rujutashinde) [reply tests/functional/test_webhook_notification.py]: done

File: tests/functional/test_webhook_notification.py (hunk 3 above)
Comment (rujutashinde) [reply tests/functional/test_webhook_notification.py]: done

**Pull request merged**